rel-eng/custom/custom.py

```diff
@@ -195,6 +197,18 @@ def _fetch_local(self):
 
         return gitrev
 
+    def _fetch_prebuilt_local(self):
+        source_files = [os.path.expanduser(source_file) for source_file in self.builder.args['source_files']]
+        dest_dir = os.path.join(self.builder.rpmbuild_sourcedir, 'archive')
+        if not os.path.exists(dest_dir):
+            os.mkdir(dest_dir)
+
+        for src in source_files:
+            debug("Copying %s to %s" % (src, os.path.join(dest_dir, os.path.basename(src))))
+            shutil.copy(src, os.path.join(dest_dir, os.path.basename(src)))
+
+        return ""
```

Comment:
What does an empty string mean here? Do we need a return value?

Reply:
Other `_fetch*` methods return a gitrev string for source, but since we're just pointing to a file we don't have any kind of hash, so empty string. That gets concatenated into the release which results in just a timestamp.